tensorflow/training/docker/2.1.1/py3/Dockerfile.cpu

```diff
@@ -108,7 +108,7 @@ RUN mkdir -p /root/.ssh/ \
 WORKDIR /
 
 RUN ${PIP} --no-cache-dir install --upgrade \
-    pip \
+    pip==20.2.1 \
```

Comment:
Why is this necessary?

Reply:
Using pip newer than 20.2.1 leads to a failure of horovod deployment

Reply:
nit: If there is another iteration, please add a comment about this to the dockerfile to make it more clear why this was done.